### ISSUE TITLE
Tapping on an image resets zoom

### ISSUE THREAD

**Issue by seanthegeek** [nextcloud/android#6797]
### Steps to reproduce
1. Open an image in the built-in browser
2. Zoom into part of the image
3. Tap once on the image to hide the UI

### Expected behaviour
- The UI should hide without resetting the zoom/panning

### Actual behaviour
- The UI hides, but the image zoom/panning resets to its original state 
- Oddly, after this happens once, the issue dies not reoccur until another image is loaded

### Can you reproduce this problem on https://try.nextcloud.com?
Not a server issue

### Environment data
Android version: 10

Device model: Pixel 3 XL

Stock or customized system: Stock

Nextcloud app version: 3.13

**Comment by tobiasKaminsky** [comment]
No idea what is happening here.
https://github.com/nextcloud/android/blob/15dcad229c2894ce1ef11e3ecf39b4791692d0c4/src/main/java/com/owncloud/android/ui/preview/PreviewImageActivity.java#L508-L527

seems to be the affected code, but there we did not changed anything over the last releases.

**Comment by seanthegeek** [comment]
Right. This bug has existed for years. I'm just now reporting it 😄

**Comment by tobiasKaminsky** [comment]
https://github.com/chrisbanes/PhotoView/issues/628

only "solution" seems to be to use a different lib, but which one…?
